.github/workflows/publish-spm-release.yaml

```diff
@@ -0,0 +1,17 @@
+name: Publish SPM Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish-spm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Assign Tag Number to RELEASE_VERSION environment variable
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Dispatch Release Workflow on ionic-team/capacitor-swift-pm
+        run: gh workflow run dispatch-release.yaml -f release-version=${{ env.RELEASE_VERSION }} --repo ionic-team/capacitor-swift-pm
+        env:
+          GITHUB_TOKEN: ${{ secrets.CAPICTOR_PUBLISH_XCFRAMEWORK }}
```

Comment:
@giralte-ionic is this correct? I just copy-pasta'd the name from our DM, just want to double check